utils/gas-utils/gas_utils.go

```diff
@@ -8,12 +8,12 @@ import (
 	"math/big"
 
 	"github.com/ava-labs/avalanchego/utils/math"
+	"github.com/ava-labs/subnet-evm/precompile/contracts/warp"
 )
 
 const (
-	ReceiveCrossChainMessageStaticGasCost           uint64 = 2_000_000
-	ReceiveCrossChainMessageGasCostPerAggregatedKey uint64 = 1_000
-	ReceiveMessageGasLimitBufferAmount              uint64 = 100_000
+	ReceiveCrossChainMessageStaticGasCost uint64 = 300_000 + warp.GasCostPerSignatureVerification
```

Comment:
The comment on the PR lists `500_000` as your calculation. How do we get to that number from these numbers?

Reply:
`warp.GasCostPerSignatureVerification = 200_000`. I'll add a comment making this clear.